### ISSUE TITLE
Error: Cannot find module 'rx'

### ISSUE THREAD

**Issue by luckydonald** [RobCoIndustries/pipboycli#3]
```
mac:~ luckydonald$ pipboycli
module.js:338
    throw err;
    ^

Error: Cannot find module 'rx'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/pipboycli/lib/index.js:10:11)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
```

**Comment by rgbkrk** [comment]
Definitely reproducible on 4.x, ran this in a fresh docker container. :sob:

If I upgrade `npm`, no problem:

```
$ docker run --net=host --rm -it node:4.1 bash
root@default:/# npm --version
2.14.4
root@default:/# npm install -g pipboycli > /dev/null 2>&1
root@default:/# pipboycli
module.js:338
    throw err;
    ^

Error: Cannot find module 'rx'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/pipboycli/lib/index.js:10:11)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
root@default:/# npm install -g npm > /dev/null 2>&1
root@default:/# npm install -g pipboycli > /dev/null 2>&1
root@default:/# pipboycli
... yay display
```

**Comment by rgbkrk** [comment]
I'm a dolt. I used `rx` in here. I am really resting on my laurels with npm3. Thank you for your patience @luckydonald.

**Comment by rgbkrk** [comment]
Shipped a new release with this added in (`v0.0.4`).
